rsconnect/main.py

```diff
@@ -1816,7 +1816,6 @@ def _write_framework_manifest(
 
     with cli_feedback("Checking arguments"):
         entrypoint = validate_entry_point(entrypoint, directory)
-        extra_files = validate_extra_files(directory, extra_files)
```

Comment:
Can you explain the approach being taken here? Is this validation being done at a later stage in the bundle creation/manifest writing?

Reply:
This change prevents extra_files from being processed twice. Once here (now deleted) and another time in https://github.com/rstudio/rsconnect-python/blob/2adcec54dc8bbb1bea77f2a1967510ec69044919/rsconnect/bundle.py#L1804